### PR TITLE
GlobalNet

### PR DESCRIPTION
**Pull request by RemiDelaunay**
ref #31 Adding affine transformation option

### CODE REVIEW

**Comment by YipengHu** [comment]
@RemiDelaunay  I prove this now but the mode build still needs updated - it could well be in your plan ;)

File: deepreg/model/layer.py [hunk 1]
```diff
@@ -1,451 +1,458 @@
...
-
-        inputs_nonskip, inputs_skip = inputs[0], inputs[1]
-        h0 = self._deconv3d_block(inputs=inputs_nonskip, training=training)
-        if self._use_additive_upsampling:
-            h0 += self._additive_upsampling(inputs=inputs_nonskip)
-        r1 = h0 + inputs_skip
-        r2 = self._conv3d_block(inputs=h0, training=training)
-        h1 = self._residual_block(inputs=[r2, r1], training=training)
-        return h1
+import tensorflow as tf
```
Comment (mathpluscode) [inline deepreg/model/layer.py:1]: I think some spaces have been added to this file?

**Comment by RemiDelaunay** [comment]
@YipengHu I thought it would be redundant to create a separate build mode for the global network when compared to the ddf one (the regularisation term perhaps but its weight can just be set to 0 in the config file).

File: deepreg/model/layer.py (hunk 1 above)
Comment (YipengHu) [reply deepreg/model/layer.py:1]: is it a problem?

Comment (ucl-candi) [reply deepreg/model/layer.py:1]: You can delete them next time you merge 😊

**Comment by YipengHu** [comment]
One might also want to output the transform parameters theta?

Yipeng
________________________________

**Pull request merged**